modules/dropdown/main/theme.scss

```diff
@@ -2,4 +2,5 @@
   background: $background-col;
   box-shadow: 0px 2px 2px 2px $shadow-col;
   border-color: $border-col;
+  margin-top: $spacing * 1px;
```

Comment:
Yeah, default spacing in the styles, then it overrides it if provided in the options when creating the dropdown. Kept the calculation of the dropdown position a bit simpler